### ISSUE TITLE
Though this site has HTTPS, the HTTP version does not actively redirect to the HTTPS version

### ISSUE THREAD

**Issue by BenG-NS** [HTTPSChecker/releases#11]
The site I'm testing forces all urls to be https, but I still see this error: Though this site has HTTPS, the HTTP version does not actively redirect to the HTTPS version.

What does this error actually mean?  There isn't much documentation available, so I'm not entirely certain what this error refers to.

**Comment by stilliard** [comment]
Hi @BenG-NS ,

The error normally shows up if you can access the homepage of the site over HTTP without it sending a redirect/location header to send it to the HTTPS version. 

Would you be able to share the URL of the site that you're seeing this on?

**Comment by BenG-NS** [comment]
Hi @stilliard ,
Unfortunately, I can't share the URL at this point.  The issue did start appearing sometime since I downloaded the installer on 3/6.  When I ran the scan shortly afterwards, this error didn't appear.  But subsequent scans returned the "Though this site has HTTPS, the HTTP version does not actively redirect to the HTTPS version" error message.  I know the app did update at least once since I downloaded it; maybe a subsequent version has changes that might have given rise to the error?

**Comment by stilliard** [comment]
Ok sure, It's possible the message has started to show more from our latest update, we did fix a number of minor issues that may have prevented the message from showing properly before, or it might be there's a issue where it's showing when it shouldn't, though without an example url of this it may be tricky to track down. 

I've tried a number of sites and the only way I can see this message is if the does in fact have HTTPS but redirects back to HTTP for the homepage, are you sure this isn't the case for the site you're testing? 

I'll keep an eye out for this appearing though, thank you for the report.
